src/webextension/background/datastore.js

```diff
@@ -3,45 +3,23 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 import DataStore from "lockbox-datastore";
+import jose from "node-jose";
 
-export const datastore = DataStore.create();
-
-export const fakeStore = {
-  _nextId: 0,
-  _datastore: new Map(),
-
-  unlock() {
-    return Promise.resolve();
-  },
-
-  add(details) {
-    const id = (this._nextId++).toString();
-    const item = {...details, id};
-    this._datastore.set(id, item);
-    return Promise.resolve(item);
-  },
-
-  update(item) {
-    if (!this._datastore.get(item.id))
-      throw new Error("item does not exist");
-    this._datastore.set(item.id, item);
-    return Promise.resolve(item);
-  },
+const keys = {
+  "encrypted": "eyJhbGciOiJQQkVTMi1IUzI1NitBMTI4S1ciLCJraWQiOiJXa3hjLUUyY0h1SDdwS244cDFDdVdKUmd1SlFiMzhSZkZKdmk3Q3VJMVg4IiwicDJjIjo4MTkyLCJwMnMiOiJQRXVSRkR3dkJIbUZ1NjA1amdFUGtSU05KTkdranpEaEVBWTdoUHc2RW9rIiwiZW5jIjoiQTI1NkdDTSJ9.mF_TyKRmlV2yL1HqFMd8GWtAw94Kl6DOd9LX_Di1d_g8AqNIJLSVaQ.AUs8cZ7P42AeyGQ0.2Uw.PA0_79GRBG1NGhiLYlaKwQ"
+};
 
-  remove(id) {
-    this._datastore.delete(id);
-    return Promise.resolve();
-  },
+let master = {
+  "kty": "oct",
+  "kid": "Wkxc-E2cHuH7pKn8p1CuWJRguJQb38RfFJvi7CuI1X8",
+  "k": "cmVzaW4gcGVjY2FkaWxsbyBjYXJ0YWdlIGNpcmN1bW5hdmlnYXRlIGFyaXRobWV0aWMgcmV2ZXJlbnRpYWw"
+};
 
-  get(id) {
-    const item = this._datastore.get(id);
-    if (item)
-      return Promise.resolve(item);
-    return Promise.reject();
-  },
+master = jose.util.base64url.decode(master.k).toString("utf8");
```

Comment:
Fair warning: this changes with lockbox-datastore#14 (-:

On the plus side, it should be a lot easier to deal with.